app/components/CheckURLBanner/index.tsx

```diff
@@ -22,18 +23,23 @@ export const CheckURLBanner: FC<Props> = ({ onHide }) => {
       <div className={styles.banner}>
         <div className={styles.banner_text}>
           <Text t="h4" align="center">
-            ⚠️ Verify the URL and bookmark this page!
+            <Trans id="checkurlbanner.verify_url_and_bookmark">
+              ⚠️ Verify the URL and bookmark this page!
+            </Trans>
           </Text>
           <Text t="caption" color="lighter" align="center">
-            <strong>dapp.klimadao.finance</strong> is the only official domain.
+            <strong>dapp.klimadao.finance</strong>{" "}
+            <Trans id="checkurlbanner.is_the_only_official_domain">
+              is the only official domain.
```

Comment:
I don't think all translations will follow the format of `<noun> is`
so we should wrap the whole thing in a tag right?

Reply:
You are right.
I also put a comment to help translators